functions/index.js

```diff
@@ -20,3 +24,21 @@ exports.checkForUser = functions.https.onRequest(async (request, response) => {
   });
 });
 
+/** This function retrieves all program details based on the requested program */
```

Comment:
```suggestion
/** This function retrieves all group details based on the requested program */
```
As discussed, change every instance of `program` to the name `group`